### PR TITLE
master: terraform: update to 0.6.12-ens.2, including upstream improvements

### PR DESCRIPTION
**Pull request by josephholsten**

### CODE REVIEW

**Comment by mlrobinson** [comment]
LGTM, by @mlrobinson

File: terraform.rb [hunk 1]
```diff
@@ -143,16 +60,21 @@ def install
     end
 
     cd terrapath do
-      system "go", "test", "./..."
+      terraform_files = `go list ./...`.lines.map { |f| f.strip unless f.include? "/vendor/" }.compact
+      # v0.6.12 - source contains tests which fail if these environment variables are set locally.
+      ENV.delete "AWS_ACCESS_KEY"
```
Comment (thejandroman) [inline terraform.rb:65]: Nice

**Comment by thejandroman** [comment]
:8ball:

**Comment by rowleyaj** [comment]
LGTM

**Pull request merged**